test/lib/sql.js

```diff
@@ -0,0 +1,203 @@
+
+const { expect } = require('chai');
+const SQL = require('../../lib/sql');
```

Comment:
Is it possible to use absolute reference instead of relative?

Reply:
abs import can be more maintainable over relative. I don't if [this](https://medium.com/@davidjwoody/how-to-use-absolute-paths-in-react-native-6b06ae3f65d1) will work here, but it works in react native. Anyway this is just a small issue.

Reply:
that method works but introduces additional complexity, this project is quite small, so we can skip using that